ui/library/src/components/DownloadButton.tsx

```diff
@@ -35,9 +39,9 @@ export const DownloadButton: React.FunctionComponent<Props> = ({
       disabled={fetching}
       onClick={() => download()}
       aria-label="Download PDF"
-      className="reader__download-btn"
+      className={classname ? `reader__download-btn ${classname}` : 'reader__download-btn'}
       {...extraProps}>
-      Download
+      {icon ? icon : 'Download'}
```

Comment:
I'd suggest renaming `icon` to `children` because that's how it's being used- whoever is using the library *could* pass an icon here, but they don't have to. Any node they pass will be rendered as a child of this button.

Reply:
@carolinepaulic okey got it